misc/python/materialize/mzexplore/extract.py

```diff
@@ -17,15 +17,18 @@
 from contextlib import closing
 from pathlib import Path
 
+import pandas as pd
```

Comment:
This is a big dependency---are we using it for more than the sum computation below? (Can everyone install pandas from a binary, or will some folks at MZ have to compile from source to use this?) I can't imagine we have more than a couple hundred entries in the arrangement sizes data frame.

Reply:
> This is a big dependency---are we using it for more than the sum computation below?

Yes, but it's not a new dependency—I added it more than 2 years ago with #9445 and we've been carrying this ever since. We are using this in `optbench` for sure and I think for some other feature benchmark computations.